f8a_worker/workers/recommender.py

```diff
@@ -702,8 +736,15 @@ def execute(self, parguments=None):
                                  .format(parguments.get('external_request_id', ''),
                                          set(companion_packages).difference(set(filtered_list))))
 
+                    # Apply License Filters
+                    lic_filtered_companions, lic_filtered_list = apply_license_filter(filtered_comp_packages_graph, input_stack)
+
+                    _logger.info("Companion Packages further filtered (based on licenses) for external_request_id {} {}"
```

Comment:
This line is too long.

Reply:
Yes, that line is very long. I have now tried best to reduce the length to whatever is currently there.